packages/strapi-design-system/src/IconButton/IconButton.tsx

```diff
@@ -54,7 +54,7 @@ export const IconButton = React.forwardRef<HTMLButtonElement, IconButtonProps>(
       <IconButtonWrapper
         aria-disabled={disabled}
         background={disabled ? 'neutral150' : undefined}
-        borderWidth={noBorder ? 0 : undefined}
+        borderWidth={noBorder ? '0px' : undefined}
```

Comment:
do you know why we have to pass a string? is it because we're just checking for a false value?

Reply:
after checking again I think it comes from the `BaseButton`, checking `borderWidth` as a falsy value
```js
if (borderColor && !borderStyle && !borderWidth) {
  return `1px solid ${theme.colors[borderColor]}`;
}
```

Reply:
What do you think about changing that to be 

```js
if (borderColor && !borderStyle && typeof borderWidth === "undefined") {
  return `1px solid ${theme.colors[borderColor]}`;
}
```

Which _should_ also solve the issue for us and therefore allow us to pass numerical values like `0`?

Reply:
sounds better indeed